examples/allLogLevels.cpp

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file allLogLevels.cpp
```

Comment:
I'm not quite sure if Arduino IDE will complain about location of this file: AFAIK an example ino sketch must be located under the folder which name is the same as the sketch. 
The name of this sketch should be ```AllLogLevels.ino``` and it should be in ```AllLogLevels`` directory.

However now I see that the file is a *.cpp file and also has a ```main()``` method.

Reply:
The cpp files are not for the Arduino IDE, and they do not show up in the examples listed in the IDE file menu. I'm actually still working on this because of CMake... The cpp files are for Linux/PC/PicoSDK. I'm still trying to find a way that avoids having an examples folder for Linux, an examples folder for arduino, and another examples folder for the PicoSDK (like it is with the other more platform specific RF24* libs).